src/utils/RecognizeFileExtension.js

```diff
@@ -13,7 +13,8 @@ export default (UrlOfAFile) => {
     fileExtension === "png" ||
     fileExtension === "ico" ||
     fileExtension === "jpe" ||
-    fileExtension === "gif"
+    fileExtension === "gif" ||
+    fileExtension === "bmp"
```

Comment:
nice

Reply:
The gif file have a funny behavior but it work sooo ... i let it there for fun but in a future release we need to take care of this behavior